PowerShellEditorServices.build.ps1

```diff
@@ -179,7 +186,7 @@ task TestProtocol -If { !$script:IsUnix} {
 task TestHost -If { !$script:IsUnix} {
     Set-Location .\test\PowerShellEditorServices.Test.Host\
     exec { & $script:dotnetExe build -c $Configuration -f net452 }
-    exec { & $script:dotnetExe xunit -configuration $Configuration -framework net452 -verbose -nobuild -x86 }
```

Comment:
Also, I have no idea what the consequences of this change could be. Because I'm not sure why that was there originally.

Reply:
I suspect it's because VSCode use to only be 32bit but now it has a 64bit version so we can probably safely remove this.